### DISCUSSION TITLE
Can't show CVAT task web page

### DISCUSSION THREAD

**Discussion by rogerhsu1121** [cvat-ai/cvat#3042] [General]
I try to run CVAT on ubuntu 16.04
Due to I have a jenkins task occupy 8080 port
so i modified docker-compose.yml
change port as 8888
    ports:
      - '8888:80'

then I run cvat on docker
sudo docker-compose up  -d

I can visit
http://localhost:8888/admin

but it show empty page when I visit http://localhost:8888 or http://localhost:8888/task/
Is there anything I missing?

**Comment by azhavoro** [comment]
Have you changed the port for the `cvat` service? If yes try to change the port for the `cvat_proxy` instead. https://github.com/openvinotoolkit/cvat/blob/develop/cvat/apps/documentation/faq.md#how-to-change-default-cvat-hostname-or-port

**Reply by rogerhsu1121** [reply]
Thanks your response.
Yes, I change the port value in docker-compose.yml directly but this issue still happen if I used localhost in server side, then I try to set CVAT_HOST as server IP and connect CVAT from another computer, it can connect to task page normally. 
It might something wrong in my server side Chrome.
thanks.
  cvat_proxy:
    container_name: cvat_proxy
    image: nginx:stable-alpine
    restart: always
    depends_on:
      - cvat
      - cvat_ui
    environment:
      CVAT_HOST: 10.57.55.121
    ports:
      - '8888:80'
